src/AspNetCore.Identity.MongoDB/MongoUserStore.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace AspNetCore.Identity.MongoDB
```

Comment:
all the formatting changes to this file is unnecessary in context of this PR. Can you revert these changes if possible?